web/client/plugins/GeoStoryNavigation.jsx

```diff
@@ -52,6 +54,7 @@ const GeoStoryNavigation = ({
             currentPage={currentPage}
             currentPosition={currentPosition}
             totalItems={totalItems}
+            geostoryId={geostoryId}
```

Comment:
This should also be removed based on this https://github.com/geosolutions-it/MapStore2/pull/5767#discussion_r470463141